Packages/manifest.json

```diff
@@ -2,12 +2,12 @@
   "dependencies": {
     "com.getfilta.artist-unityplug": "https://github.com/getfilta/artist-unityplug.git",
     "com.unity.ide.vscode": "1.2.4",
-    "com.unity.nuget.newtonsoft-json": "2.0.2",
-    "com.unity.render-pipelines.universal": "11.0.0",
+    "com.unity.nuget.newtonsoft-json": "2.0.1-preview.1",
```

Comment:
Why the newtonsoft downgrade?

Reply:
i didn't actively do this. i think when i upgraded unity, it auto-did this.